release/models/system/openconfig-system.yang

```diff
@@ -47,7 +48,12 @@ module openconfig-system {
     Section 4.c of the IETF Trust's Legal Provisions Relating
     to IETF Documents (http://trustee.ietf.org/license-info).";
 
-  oc-ext:openconfig-version "0.17.1";
+  oc-ext:openconfig-version "0.17.2";
+
+  revision "2023-08-28" {
+    description "Add leaf for referencing a hashing policy.";
+    reference "0.17.2";
```

Comment:
Given that this is implemented at this point, this module should be `1.x.y.` at least. This also should not be a patch change.

Reply:
I marked the version to `1.18.0`.